Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -36,21 +36,21 @@ public QuestionsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         ///Test to get all Questions 
         /// </summary>
         /// <returns></returns>
-        [Fact]
-        public async Task GetAllQuestionsAsyncTest()
-        {
-            string userName = "sandipan@promactinfo.com";
+        //[Fact]
+        //public async Task GetAllQuestionsAsyncTest()
+        //{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)